charts/apisix/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,77 @@
+{{- if .Values.apisix.networkPolicy.enabled }}
+{{- $fullName := include "apisix.fullname" . -}}
+kind: NetworkPolicy
+apiVersion: {{ template "networkPolicy.apiVersion" . }}
+metadata:
+  name: {{ include "apisix.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: 
+    {{- include "apisix.labels" . | nindent 4 }}
+  {{- if .Values.apisix.networkPolicy.annotations }}
+    {{- with .Values.apisix.networkPolicy.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
+  {{- end }}
+spec:
+  podSelector:
+    matchLabels: {{- include "apisix.selectorLabels" . | nindent 6 }}
+  policyTypes:
+    - Ingress
+  {{- if .Values.apisix.networkPolicy.additionalEgress }}
+    - Egress
+  egress:
+    - ports:
+        - port: 53
+          protocol: UDP
+    - ports:
```

Comment:
The egress rules are used to limit the destinations that APISIX can reach, so here use APISIX ports as the "target port" is pathological for me.

Reply:
Yes, sorry my bad. It was clearly a mistake. 
What about now ? Is the `egress` policy OK for you ?

Reply:
I think we don't have to restrict the source port in the egress rules.

Reply:
So, should we remove all the egress port definition and let the user choose whatever ports in the egress they might want ?

Reply:
I think specify the source port is meaningless, as the source ports usually specified by the kernel, it's somewhat random for users.